composer.json

```diff
@@ -1,24 +1,22 @@
 {
     "name": "jordanbrauer/unit-converter",
-    "type": "component",
```

Comment:
`type: component` does not exist - changed to `type: library` as per https://getcomposer.org/doc/04-schema.md#type